docs/topics/logging.txt

```diff
@@ -650,13 +687,24 @@ logging module.
 Django's default logging configuration
 ======================================
 
-By default, Django configures the ``django.request`` logger so that all messages
-with ``ERROR`` or ``CRITICAL`` level are sent to :class:`AdminEmailHandler`, as
-long as the :setting:`DEBUG` setting is set to ``False``.
+By default, Django configures the following logging:
+
+When :setting:`DEBUG` is ``True``:
+
+* The ``django`` catch-all logger sends all messages at the ``INFO`` level or
+  higher to the console. Django doesn't make any such logging calls at this
```

Comment:
Oh, I see - it's "at ``INFO`` level or higher" - Django's internal logging in those modules is at ``DEBUG`` level. I think that should be clarified.